src/manifold/src/impl.h

```diff
@@ -17,14 +17,33 @@
 
 #include "manifold/collider.h"
 #include "manifold/manifold.h"
-#include "manifold/optional_assert.h"
 #include "manifold/polygon.h"
 #include "manifold/sparse.h"
-#include "manifold/utils.h"
 #include "manifold/vec.h"
-#include "quickhull.h"
 #include "shared.h"
 
+#if defined(__GNUC__)
+
+#define IGNORE_DEPRECATED_BEGIN  \
+  _Pragma("GCC diagnostic push") \
+      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
```

Comment:
This is cool, but should we just go ahead and remove `Mesh` for v3.0? It would be a lot less maintenance burden.

Reply:
true